### DISCUSSION TITLE
brodalink rm4 pro programmare telecomando ls&motion 1 channell

### DISCUSSION THREAD

**Discussion by marcam1974** [indomus/forum#1257] [HUB: Home Assistant]
salve a tutti sono nuovo del forum. piacere e buon anno.
Ho il seguente problema, ho delle barre led nella gola della cucina ma non riesco a programmare il telecomando che e' marcato ls&motion 1 channel con broadlink. Non credo che si tratti di un telecomando rolling code, non ho capito dalle specifiche se si tratta di un telecomando a RF o IF.
Magari qualche d'uno e' in grado di aiutarmi. 
Grazie mille per la collaborazione.

**Comment by indomus** [comment]
> salve a tutti sono nuovo del forum. piacere e buon anno. Ho il seguente problema, ho delle barre led nella gola della cucina ma non riesco a programmare il telecomando che e' marcato ls&motion 1 channel con broadlink. Non credo che si tratti di un telecomando rolling code, non ho capito dalle specifiche se si tratta di un telecomando a RF o IF. Magari qualche d'uno e' in grado di aiutarmi. Grazie mille per la collaborazione.

Buon anno a te! Beh ovviamente prima capisci se è RF o IR. Per farlo ti basta provare a comandarlo da dietro un ostacolo (un ostacolo vero, tipo un muro), se funziona uguale è sicuramente RF

**Comment by marcam1974** [comment]
e' un telecomando RF ma il broadlink non riesce a captare il segnale, mi dice failed. Strano perché non penso sia Rolling Code.
Qualche d'uno ha avuto lo stesso problema?
